src/scripts.js

```diff
@@ -15,6 +20,7 @@ const searchError = document.querySelector('.search-error');
 const dropdownSelection = document.querySelector('#tag-selector');
 const goButton = document.getElementById('go');
 const goListButton = document.getElementById('goListButton');
+const goFavoritesButton = document.getElementById('goFavoritesButton');
```

Comment:
Now I'm wondering if we should pull this search block out and put it into its own wrapper, and just hide it when we don't need it.